.github/workflows/build.yml

```
@@ -87,7 +102,7 @@ jobs:
           if-no-files-found: error
 
       - uses: actions/upload-artifact@v4
-        if: ${{ matrix.arch == 'arm' && matrix.api-version == 5.5 }}
+        if: ${{ matrix.arch == 'arm' && matrix.api-version == 6.0 }}
```

arm and 6.0 are both strings but the usage is not consistent.

```suggestion
        if: ${{ matrix.arch == 'arm' && matrix.api-version == '6.0' }}
```

---

Thank you for check. I also modified other lines.